### PR TITLE
KD Edit article revamp

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Fixed issue with sourceURL detected in https://github.com/liferay-lima/liferay-portal/pull/3540#issuecomment-1263706664

Screenshot 2022-10-03 at 15 23 37

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:knowledge-base

File: modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp [hunk 1]
```diff
@@ -161,7 +161,7 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 						</c:if>
```
Comment (AliciaGarciaGarcia) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp:161]: the same code as sourceURL will be needed for the description field.

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:knowledge-base

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:reevaluate:1387401_4086

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Review by boton — APPROVED** [review]
LGTM. Much more beautiful than our previous interface, thanks @ambrinchaudhary 
I added a little thing to removing event listeners on portlet dispose 162c453e39450758394dac5ce59f52d1e589792a.

**Comment by austinchiang** [comment]
**Reviewing...** :eyes:

**Comment by austinchiang** [comment]
https://github.com/liferay-lima/liferay-portal/pull/3562

**Pull request closed without merging**